### PR TITLE
Release 2.29.0

### PR DESCRIPTION
**Pull request by rfalgout**
See issue #935

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -7,6 +7,22 @@
...
 
+Version 2.29.0 released 2023/06/22
+
+- Added ILU support for AMD GPUs
+- Added new error handling feature to print messages to a memory buffer
+- Enabled more support with SYCL: MGR, AMS/ADS/ADE, AIR, ...
+- Added MAGMA support
+- Added support for CUDA 12
+- Added HYPRE_Finalized function
+- Added new MGR features: Non-Galerkin options, block diagonal interpolation,
```
Comment (victorapm) [inline CHANGELOG]:
```suggestion
- Ported MGR features to CUDA and HIP: Non-Galerkin options, block diagonal interpolation, and block Jacobi relaxation
```

File: CHANGELOG [hunk 2]
```diff
@@ -7,6 +7,22 @@
...
 # =============================================================================
 
+Version 2.29.0 released 2023/06/22
+
+- Added ILU support for AMD GPUs
+- Added new error handling feature to print messages to a memory buffer
+- Enabled more support with SYCL: MGR, AMS/ADS/ADE, AIR, ...
+- Added MAGMA support
+- Added support for CUDA 12
+- Added HYPRE_Finalized function
```
Comment (victorapm) [inline CHANGELOG]:
```suggestion
- Added HYPRE_[Initialized,Finalized] functions
```

**Review by victorapm — APPROVED** [review]
Thanks, Rob! I just suggested some minor changes

**Review by ulrikeyang — APPROVED** [review]
Looks good with Victor's changes. Thank you.

**Review by waynemitchell — APPROVED** [review]
Looks good. Thanks!

**Review by osborn9 — APPROVED** [review]
Nice work all! Looks good!

**Review by liruipeng — APPROVED** [review]

**Pull request merged**